src/cljs/rems/application.cljs

```diff
@@ -594,6 +594,47 @@
              [:option {:value key}
               (get label language key)]))]))
 
+(defn- encode-multiple-values [values]
+  (->> values
+       sort
+       (str/join " ")))
+
+(defn- decode-multiple-values [value]
+  (-> value
+      (str/split #"\s+")
```

Comment:
Isn't this logic coupled with the normalizer? Should it be in one place and tested together?